android/app/src/main/java/org/fossasia/openevent/utils/SmoothActionBarDrawerToggle.java

```diff
@@ -23,6 +24,16 @@ public SmoothActionBarDrawerToggle(Activity activity, DrawerLayout drawerLayout,
     }
 
     @Override
+    public void onDrawerOpened(View drawerView) {
```

Comment:
Umm, why are you adding these methods, if you are just calling super inside them?
I can't see these methods being overridden in the code that we already have.